tutorials/natarajan2022-extraction.md

```diff
@@ -0,0 +1,241 @@
+# Extraction for Natarajan et al. (2022)
+
+[Natarajan et al. (2022)](https://doi.org/10.1016/j.medj.2022.04.001) quantified fecal shedding of different SARS-CoV-2 gene targets in feces for 113 individuals, and their data are published across several files in a [GitHub repository](https://github.com/alex-dahlen/lambda_fecal_shedding/). For reproducibility, we consider commit `7affa71`. They obtained diverse data using different sample collection kits, different gene targets, and different quantification methods.
+
+```python
+import pandas as pd
+import yaml
+import shedding_hub as sh
```

Comment:
What's this "shedding_hub"?

Reply:
The file `shedding_hub/__init__.py` defines a Python package that we can use to store common functionality. Currently, the package only contains a utility function to normalize strings (see `shedding_hub/util.py`). We could add, for example, the `folded_str` and `literal_str` functionality you introduced so it can be used across different data processing scripts. 

At a later point, we may want to release this Python so it can be installed as `pip install shedding_hub` to provide utility functions for loading data.

Reply:
I like this idea to have all common functions in a package and we can import all of them simultaneously. Thank you for the explanation.